### PR TITLE
Update code of `8-subscription` with new version.

### PR DESCRIPTION
**Pull request by Youhandan**
I have updated `8-subscription` according to `5-authentication` you updated. However I got the error of 'WebSocket connection to '<URL>' failed: WebSocket is closed before the connection is established.' . 

So my page do not update when server data is updated. I can not figure out it but it may not be related to the code I updated.

Please review the code and see if it work well in your local. If you can work, it is not the code problem and could you help me figure out my problem?

### CODE REVIEW

File: content/frontend/vue-apollo/8-subscriptions.md [hunk 1]
```diff
@@ -58,24 +58,25 @@ const wsClient = new SubscriptionClient('__SUBSCRIPTION_API_ENDPOINT__', {
...
 })
 
-const networkInterfaceWithSubscriptions = addGraphQLSubscriptions(
-  networkInterface,
+const authMiddleware = new ApolloLink((operation, forward) => {
+//add the authorization to the headers
+  const token = localStorage.getItem(GC_AUTH_TOKEN);
+  operation.setContext({
+  headers: {
+  		authorization: token ? `Bearer ${token}` : null
```
Comment (probil) [inline content/frontend/vue-apollo/8-subscriptions.md]: too much identation here. And the next line as well

File: content/frontend/vue-apollo/8-subscriptions.md [hunk 2]
```diff
@@ -190,6 +190,8 @@ Still in `src/components/LinkList.vue` implement `updateQuery` like so:
 
 ```js(path=".../hackernews-vue-apollo/src/components/LinkList.vue")
 updateQuery: (previous, { subscriptionData }) => {
+if (!subscriptionData.data.Link) return
```
Comment (probil) [inline content/frontend/vue-apollo/8-subscriptions.md]: Add 2 spaces here (better indentation)

File: content/frontend/vue-apollo/8-subscriptions.md [hunk 3]
```diff
@@ -264,6 +266,8 @@ Open `src/components/LinkList.vue` and add the following object to the `sunscrib
 {
   document: NEW_VOTES_SUBSCRIPTION,
   updateQuery: (previous, { subscriptionData }) => {
+  	if (!subscriptionData.data.Vote) return
```
Comment (probil) [inline content/frontend/vue-apollo/8-subscriptions.md]: Remove spaces at the start so it align with `const` on the next line (indentation)

**Review by probil — CHANGES_REQUESTED** [review]
Fix indentation please and I will merge

**Comment by Youhandan** [comment]
fixed the indention problem.

**Pull request merged**